### DISCUSSION TITLE
Multipl LDAP providers

### DISCUSSION THREAD

**Discussion by snburkitt** [requarks/wiki#3848] [Help / Questions]
I have added my Active Directory domain as an Authentication Strategy and it works great, AD users are able to login.

I have another office that uses its own AD domain. I have added their Domain as a second AD strategy and those user are also able to login. 

However, now usres from the first domain are getting "Invalid Username/password" error. It appears that logging into the second Domain is breaking the link to the first. 

Is there any way to fix this? Is it supported to have two AD domains listed?

**Comment by bdp13csail** [comment]
I also have a similar situation. I wrote a custom authentication module.
I am using the module as 2 instances like the reporter mentioned, with separate configuration. One instance of the module has configuration A and a second instance has configuration B.

When users log in, only configuration A from the first module is passed in no matter which one is chosen. 
In code I can verify this, if I console log a property, I can see the property value is always from configuration A.
